tests/api2/test_305_container.py

```diff
@@ -136,36 +143,422 @@ def test_10_get_new_public_image_id(request):
+    assert {'name': 'Eric_Var', 'value': 'Something'} in config['containerEnvironmentVariables'], results.text
+
+
+@pytest.mark.dependency(name='hostpath-mountpath')
+def test_36_create_datasets_hostpath_and_mountpath(request):
+    depends(request, ['pool_04'], scope='session')
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-hostpath'})
+    assert result.status_code == 200, result.text
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-mountpath'})
+    assert result.status_code == 200, result.text
+
+
+@pytest.mark.dependency(name='tc_volumes')
+def test_37_set_truecommand_ix_chart_volumes(request):
+    depends(request, ['tc_chart_release'])
+    global payload
+    payload = {
+        'values': {
+            'volume': [
+                {
+                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
+                    'datasetName': f'{pool_name}/tc-mountpath',
+                }
+            ]
+        }
+    }
+    results = PUT(f'/chart/release/id/{tc_release_id}/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_38_verify_ix_chart_config_volumes(request):
+    depends(request, ['tc_volumes'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    payload_volume = payload['values']['volume'][0]
+    assert payload_volume in results.json()['config']['volume'], results.text
+
+
+@pytest.mark.dependency(name='tc_hostPathVolumes')
+def test_39_set_truecommand_ix_chart_hostPathVolumes(request):
+    depends(request, ['tc_chart_release'])
+    global payload
+    payload = {
+        'values': {
+            'hostPathVolumes': [
+                {
+                    'hostPath': f'/mnt/{pool_name}/tc-hostpath',
+                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
+                    'readOnly': True
+                }
+            ]
+        }
+    }
+    results = PUT(f'/chart/release/id/{tc_release_id}/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_40_verify_ix_chart_config_hostPathVolumes(request):
+    depends(request, ['tc_hostPathVolumes'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    payload_hostPathVolumes = payload['values']['hostPathVolumes'][0]
+    config = results.json()['config']
+    assert payload_hostPathVolumes in config['hostPathVolumes'], results.text
```

Comment:
Okay so it's nice to see that the changes are reflected in the configuration of the chart release but i think it will be much nicer to confirm that the host path is actually being consumed by the chart release. You can verify that by looking up `resources.host_path_volumes` for the chart release in question.

Reply:
added that to the test after that one.